src/tex/ms.tex

```diff
@@ -14,17 +14,18 @@
 
 \clearpage
 \begin{abstract}
-    Large language models (LLMs) have gained widespread interest due to their ability to process human language.
-    Also, LLMs have shown promise in addressing the limited data problem in chemistry and are increasingly being harnessed to improve and extend existing workflows.
+Large language models (LLMs) have gained widespread interest due to their ability to process human language and perform tasks on which they have not been explicitly trained.
 
-    However, we possess only a limited systematic understanding of the reasoning capabilities of LLMs, which would be required to improve models and mitigate potential harms.
-    Here, we introduce \enquote{\chembench,} an automated framework for evaluating the chemical knowledge and reasoning abilities of state-of-the-art LLMs against the expertise of chemists.
+However, we possess only a limited systematic understanding of the chemical capabilities of LLMs, which would be required to improve models and mitigate potential harms.
+Here, we introduce \enquote{\chembench,} an automated framework for evaluating the chemical knowledge and reasoning abilities of state-of-the-art LLMs against the expertise of chemists.
 
-    We curated more than 2,800 question-answer pairs, evaluated leading open and closed-source LLMs, and found that, on average, the best models outperformed the chemists.
-    The models, however, struggle with some chemical reasoning tasks that are easy for the human experts and provide overconfident, misleading predictions, such as about chemicals' safety profiles.
-    Although LLMs demonstrate remarkable proficiency in chemical tasks, further research is critical to enhancing their safety and utility in chemical sciences.
+We curated more than 2,800 question-answer pairs, evaluated leading open—and closed-source LLMs, and found that the best models outperformed the best human chemists in our study on average.
+However, the models struggle with some tasks that are easy for human experts and provide overconfident predictions.
```

Comment:
```suggestion
However, the models struggle with some tasks that are easy for human experts, providing overconfident predictions.
```

Reply:
hm -  what is the motivation behind this suggestion? I felt that the "and" was here for the enumeration of two separate observations. Without "and" it seems a bit more as if "overconfident predictions" would be one of the tasks that are easy for humans

Reply:
Sounded better to me, but with the "and" sounds also good